dotnet/samples/KernelSyntaxExamples/Example85_AgentCharts.cs

```diff
@@ -54,7 +49,7 @@ public async Task CreateChartAsync()
             await InvokeAgentAsync(
                 thread,
                 "1-first", @"
-Display this data using a bar-chart:
+Display this data using a bar-chart with no sumation:
```

Comment:
```suggestion
Display this data using a bar-chart with no summation:
```

Reply:
Still working on getting a good build.  Spell checker will catch this.